test/e2e/e2e_suite_test.go

```diff
@@ -58,9 +58,17 @@ func TestBuildRun(t *testing.T) {
 var _ = SynchronizedBeforeSuite(func() []byte {
 	var err error
 
-	Logf("Starting local operator")
-	operatorCmd, err = startLocalOperator()
-	Expect(err).ToNot(HaveOccurred(), "Failed to start local operator")
+	operator := os.Getenv(EnvVarOperator)
+	switch operator {
```

Comment:
i think u can use `os.Getenv(EnvVarOperator)` here directly

Reply:
Can u check if in line 75 , the `populateOperatorSDKTestFrameworkScheme` should only run if `EnvVarOperator`  equals to `start_local` ? Im just wondering if I already have an operator started somewhere else, most probably the crds scheme were already added. Im not very sure on this.

Reply:
The `operator` variable is used again later in line 75. Will check your second comment.

Reply:
> is `populateOperatorSDKTestFrameworkScheme` needed?

Yes. The function makes the scheme of custom resource definitions known to the operator sdk test framework. Even if `TEST_E2E_OPERATOR=managed_outside`, `TEST_E2E_CREATE_GLOBALOBJECTS` could still be `true` which means that the build strategy custom resource definitions are needed to be known later in the the before suite function.